### DISCUSSION TITLE
Spread operator not acknowledged

### DISCUSSION THREAD

**Discussion by AlessandroRosa** [jquery/esprima#2117] [Q&A]
Hello !

This code is correct, but esprima detects a syntax error:

var _whatever = { ..._plugin_rec_var_vals };

**Comment by kungfooman** [comment]
When you see software is not maintained anymore, just use something else? In this case `Babel` should do it, or what binds you to `esprima`?

**Comment by AlessandroRosa** [comment]
Ok, thank you for your quick response.
I found esprima very easy and clean.

Alessandro Rosa

Il Dom 14 Ago 2022, 15:27 Hermann Rolfes ***@***.***> ha
scritto:

**Comment by Meir017** [comment]
@AlessandroRosa try using https://github.com/node-projects/esprima-next or https://github.com/acornjs/acorn both of these parsers return an AST similar to the ESTree spec

**Comment by AlessandroRosa** [comment]
I will, thank you!

Have a nice day!

Il Lun 15 Ago 2022, 05:53 Meir Blachman ***@***.***> ha
scritto:
